apps/aecore/test/aecore_oracle_test.exs

```diff
@@ -115,6 +115,36 @@ defmodule AecoreOracleTest do
     oracle = OracleStateTree.get_oracle(Chain.chain_state().oracles, oracle_key)
     assert oracle.expires == 15
 
+    Chain.clear_state()
+    register_oracle(:valid)
+    Miner.mine_sync_block_to_chain()
+    Miner.mine_sync_block_to_chain()
+    oracle_tree = Chain.chain_state().oracles.oracle_tree
+    oracle_key = oracle_tree |> PatriciaMerkleTree.all_keys() |> List.first()
+
+    oracle = OracleStateTree.get_oracle(Chain.chain_state().oracles, oracle_key)
+    expires_oracle = 12
+    assert oracle.expires == expires_oracle
+
+    Miner.mine_sync_block_to_chain()
```

Comment:
would it be possible to use smaller numbers, so we don't have to do so much `Miner.mine_sync_block_to_chain()`